.golangci.yml

```diff
@@ -28,6 +28,9 @@ issues:
     - linters:
         - revive
       text: "don't use leading k in Go names"
+    - linters:
```

Comment:
This is required otherwise we are forced to add package comments to every single package missing it, which is a lot. It's not worth adding it at this time.